cpp/log/consistent_store.h

```diff
@@ -79,6 +81,10 @@ class EntryHandle : public EntryHandleBase {
     return &entry_;
   }
 
+  bool SerializeToString(std::string* output) const override {
```

Comment:
One may wonder why this method is needed given you could simply call Entry().SerializeToString ?